File: 11025.js
const fs = require('fs');

function solve() {
	const input = fs.readFileSync('/dev/stdin').toString().split('\n');
	let inputIdx = 0;
	let [N, K] = input[inputIdx++].split(' ').map(v => +v);

	let ans = 0;

	for (let i = 2; i <= N; i++){
		ans = (ans + K) % i;
	}

	return ans + 1;
}

console.log(solve());

File: 11025.py
from sys import stdin

input = stdin.readline


def solve():
    N, K = map(int, input().split())
    ans = 0
    for i in range(2, N + 1):
        ans = (ans + K) % i
    return ans + 1


if __name__ == '__main__':
    print(solve())
